src/ResourceManagement/OperationalInsights/OperationalInsights.Test/ScenarioTests/LinkedServiceOperationsTests.Scenario.cs

```diff
@@ -0,0 +1,101 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.OperationalInsights;
+using Microsoft.Azure.Management.OperationalInsights.Models;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using OperationalInsights.Tests.Helpers;
+using System;
+using System.Linq;
+using System.Net;
+using Xunit;
+
+namespace OperationalInsights.Test.ScenarioTests
+{
+    public class LinkedServiceOperationsTests : TestBase
+    {
+        private const string resourceGroupName = "OIHyak7814";
+        private const string automationAccountName = "sdkTestAccount";
+        private const string automationAccountName2 = "testAcc";
+        private const string workspaceName = "AzTest6856";
+
+        [Fact]
+        public void CanCreateUpdateDeleteLinkedService()
+        {
+            using (MockContext context = MockContext.Start(GetType().FullName))
+            {
+                var client = TestHelper.GetOperationalInsightsManagementClient(this, context);
+                var subId = client.SubscriptionId;
+               
+                var linkedServiceName = "Automation";
+                var accountResourceIdFromat = "/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.Automation/automationAccounts/{2}";
+                var accountResourceId = string.Format(accountResourceIdFromat, subId, resourceGroupName, automationAccountName);
```

Comment:
If I understand correctly, you are saying that we should have helper methods to build the accountResourceId? Well, in real life use cases, the accountResourceId comes from another API. So the user doesn't really need to manually build this string.